config/locales/en/schedule/confirmation.yml

```diff
@@ -6,3 +6,6 @@ en:
       review_status:
         reviewed: This content has been reviewed and approved for publication.
         not_reviewed: This content needs to be published urgently but should be reviewed as soon as possible.
+      radio_not_selected:
+        title: No radio button has been selected
```

Comment:
I think the card had some different copy to use - could you double check?

Reply:
Yep didn't realise content had already been agreed upon 👍